<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerproduct</class>
 <widget class="QWidget" name="registerproduct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>238</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registro de Productos</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>200</width>
     <height>234</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pbtn_previous">
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbtn_next">
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Producto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_producto">
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Unidades</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_unidades">
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Precio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_precio">
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pbtn_add">
      <property name="font">
       <font>
        <family>Yu Gothic</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbtn_nuevo">
      <property name="font">
       <font>
        <family>Yu Gothic</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nuevo</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
